HandyImage.user.js

```diff
@@ -1,8 +1,8 @@
 // ==UserScript==
 // @name		Handy Image
-// @version		2017.02.03
+// @version		2017.02.09
 // @author		Owyn
-// @contributor	ubless607, bitst0rm
+// @contributor	ubless607, bitst0rm, LightGrey, RiZKiT, MurkBRA, kickerofelves, HalliHalloSchatz, hurda, johnmcdee, AgentRev, Lambda11, Quicksaver
```

Comment:
please don't duplicate the information already present on github, you included even my test account... if people themself would ask me to include em - I will, to thank them for their continuous support.